src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.26.0-dev",
+  "version": "13.27.0-dev",
```

Comment:
Nit: will there be updates to the spec, given that we're withholding the extrusion stuff?

Reply:
No, there will be no update to the spec in this release after we decided to withhold changes, good catch!